addon/legacy/helpers/intl-get.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2015, Yahoo! Inc.
+ * Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
+ */
+
+import Ember from 'ember';
+
+import {
+    Stream,
+    read,
+    readHash,
+    destroyStream
+} from 'ember-intl/utils/streams';
+
+// Backwards compatibility for Ember < 1.13
+export default function (value, options) {
```

Comment:
I really hate this method.. maybe I'll attempt cleaning it up once i get the tests passing